json-matchers.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |spec|
   spec.name          = "json-matchers"
   spec.version       = JSON::Matchers::VERSION
   spec.authors       = ["Sean Doyle"]
-  spec.email         = ["seandoyle@thoughtbot.com"]
+  spec.email         = ["sean.p.doyle24@gmail.com"]
   spec.summary       = %q{Validate your Rails JSON API's JSON}
   spec.description   = %q{Validate your Rails JSON API's JSON}
   spec.homepage      = "https://github.com/seanpdoyle/json-matchers"
```

Comment:
Yes! ([from this comment](https://github.com/thoughtbot/json-matchers/pull/15#issuecomment-142948993))